src/components/BottomBarCourse.vue

```diff
@@ -98,18 +98,130 @@
             <p>{{ courseObj.description }}</p>
           </div>
         </div>
+        <div class="bottombarcourse-bar-details-noOverflow" v-if="isSmallerWidth">
```

Comment:
Definitely a way to do this without duplicating all of the code. Want to get this merged in though, so could you create a Monday.com item and assign you and me to it, and one of us can cover it later?

Reply:
Monday item made!